pkg/mapper/configmap/configmap.go

```diff
@@ -115,15 +115,19 @@ func ParseMap(m map[string]string) (userMappings []config.UserMapping, roleMappi
 	rawUserMappings := make([]config.UserMapping, 0)
 	userMappings = make([]config.UserMapping, 0)
 	if userData, ok := m["mapUsers"]; ok {
-		userJson, err := utilyaml.ToJSON([]byte(userData))
+		if LenientYamlEKS {
```

Comment:
IMO one improvement would be push this logic into a utility.  We can have a `pkg/config/yaml` which includes our own implementation of yaml.Unmarshal which makes this decision based on a struct variable.  Then its set when the struct is initialized and we can do that once when authenticator starts up based on the flag value.

Reply:
> IMO one improvement would be push this logic into a utility. We can have a `pkg/config/yaml` which includes our own implementation of yaml.Unmarshal which makes this decision based on a struct variable. Then its set when the struct is initialized and we can do that once when authenticator starts up based on the flag value.

thank you for the comments. this can be a future goal to build a `pkg/config/yaml` if there are more changes about yaml parsing.

Reply:
+1 for this idea, also could make a case where any other need for it would just be able to import it from pkg

Reply:
This is only short term fix to solve certain case in EKS configmap parsing.  We are working on long term solution for this. Given essentially it is a short-term fix, I'd like to constrain it only in configmap.go. Because if exposed in a new package, we can't control how the new package will be used potentially. It is harder to clean it up in future.